plugin-server/src/main/ingestion-queues/anonymous-event-buffer-consumer.ts

```diff
@@ -79,7 +79,10 @@ export const startAnonymousEventBufferConsumer = async ({
 
             status.debug('⬆️', 'Enqueuing anonymous event for processing', { job })
             try {
-                await graphileQueue.enqueue(JobName.BUFFER_JOB, job)
+                await graphileWorker.enqueue(JobName.BUFFER_JOB, job, {
```

Comment:
I think the answer is yes as retryOnFailure is false, but this call will fail on PG failure right?

Reply:
the behavior remains the same as before in this case - i.e. if retries are enabled we don't throw, else we do. That means we will throw in the anonymous event consumer